src/inmanta/server/services/orchestrationservice.py

```diff
@@ -736,8 +736,9 @@ async def _put_version(
             if is_partial_update:
                 # Make mypy happy
                 assert partial_base_version is not None
-                rids_unchanged_resource_sets: abc.Set[
-                    ResourceIdStr
+                # This dict maps a resource id to its resource set for unchanged resource sets.
+                rids_unchanged_resource_sets: dict[
```

Comment:
This could probably be better named now that it changed types but I couldn't find a good name